blog/_posts/2016-10-10-julia-0.5-highlights.md

```diff
@@ -289,72 +289,71 @@ It's common to write expressions applying multiple vectorized functions to some
 For example, one might write something like:
 
 {% highlight julia %}
-atan2(cos(X), sin(Y))
+max(abs(X), abs(Y))
 {% endhighlight %}
 
-Currently, this allocates two temporary intermediate arrays – one to store `cos(X)` and one for `sin(Y)`.
+This computes the absolute values of each element of `X` and `Y` and takes the larger of the corresponding elements from `abs(X)` and `abs(Y)`.
+In this form, the code allocates two temporary intermediate arrays – one to store `abs(X)` and one for `abs(Y)`.
 If we use vectorized function call syntax, however, these calls are syntactically fused into a *single* call to `broadcast` with an anonymous function.
-In other words, `atan2.(cos.(X), sin.(Y))` becomes
+In other words, `max.(abs.(X), abs.(Y))` becomes
```

Comment:
Maybe put "the expression" before the expression and change "becomes" -> "is transformed to", "is lowered to"?